packages/babel-preset-cli/index.js

```diff
@@ -0,0 +1,14 @@
+module.exports = () => ({
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: '8.0.0',
```

Comment:
Node 8.9.0 is when LTS started, and notably 8.7.0 is when native async functions were added. So by supporting LTS and newer, we can use native async functions (and the latest Node 12 supports experimental async stack traces when debugging).

```suggestion
          node: '8.9.0',
```